assignments/function-conversion.js

```diff
@@ -1,23 +1,15 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
-// console.log("Function was invoked!");
-// };
+// let myFunction = function => console.log("Function was invoked!") 
```

Comment:
by putting the word function in the middle there (= function =>) you're telling this function that it has a param named function. this won't stop this particular function, but in the future when you have no parameters you'll just use empty parens. let myfunc = () =>